app/javascript/components/catalog-form/catalog-form.schema.js

```diff
@@ -0,0 +1,49 @@
+import { componentTypes, validatorTypes } from '@data-driven-forms/react-form-renderer';
+
+function createSchema(options) {
+  const fields = [{
+    component: componentTypes.SUB_FORM,
+    title: __('Basic Info'),
+    fields: [{
+      component: componentTypes.TEXT_FIELD,
```

Comment:
Oh yeah we are missing the `validateOnMount: true` property to show errors on first render. I've missed that. I guess the ruby forms don't do that.

Reply:
You guess right! 😸